tests/integration/tier1/remove-all-submgr-pkgs/test_no_submgr_pkg_installed.py

```diff
@@ -0,0 +1,34 @@
+import platform
+
+from envparse import env
+
+
+def test_no_sub_manager_installed(shell, convert2rhel):
+    """Test that no subscription manager is installed and the conversion
+    is able to get to the last point of the rollback.
+
+    """
+
+    assert shell("yum remove -y subscription-manager").returncode == 0
+
+    with convert2rhel(
+        ("--no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
+        )
+    ) as c2r:
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+        # On OracleLinux8 there is one question less than on other distros
+        if "oracle-8" not in platform.platform():
```

Comment:
If I recall correctly, centos-7 does have one less question as well.

Reply:
I checked the logs and it's ok on CentOS-7.
http://artifacts.osci.redhat.com/testing-farm/e9737651-1226-4697-a677-0c39dd0991ed/work-remove_all_submgr_pkgsnwN0V6/log.txt
It may be unreliable though. As I think it depends wherter we have installed some packages that c2r wants to remove during the pre-conversion checks.

Reply:
Ah, I see, my bad then